content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,264 @@
+---
+title: "Install and Manage MAAS Gateway"
+metaTitle: "Install and Manage MAAS Private Cloud Gateway"
+metaDescription: "Learn how to install and manage the MAAS Private Cloud Gateway in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
```

Comment:
Rita, there are four how-to guides on this page. The heading sizes need to be adjusted so that the four how-to show up on the right navigation menu. 

I think these are the top-level headings we need,

# Install the Gateway
# Update and Manage MAAS Gateway
# Delete MAAS Gateway
# Resize MAAS Gateway
# Next Steps

![CleanShot 2023-02-28 at 10 04 10](https://user-images.githubusercontent.com/29551334/221925237-8cd83a7d-e939-49a8-bcae-48a1dad5bf8c.png)

Reply:
@karl-cardenas-coding Adjusted heading sizes in this doc but left the two remaining docs as is since they are single how-tos. 
This doc originally followed our how-to style but as I understand your comment, when there are multiple how-tos that fall under the "manage" category as this one does, we should mind the right navigation menu and adjust headings.